### PR TITLE
Use r-appveyor for testing and building binary package on Windows

### PR DESCRIPTION
**Pull request by krlmlr**
* Turn on AppVeyor for this repo at https://ci.appveyor.com/projects

### CODE REVIEW

**Comment by hadley** [comment]
Is there a need for this given that testthat has no so litte c code?  I'd prefer to not use a system that I don't know anything about (and hence need to invest some time to learn)

**Comment by krlmlr** [comment]
I'm not sure if and when Travis will support Windows. [r-appveyor](https://github.com/krlmlr/r-appveyor) uses `travis-tool` from r-travis under the hood. All you need is to [sign up with AppVeyor](https://ci.appveyor.com/login) and enable testing -- setup and forget, it'll take less than five minutes.

The major benefit is that binary packages for Windows are available for download for each successful build.

**Comment by hadley** [comment]
Ok, but it if causes me problems you'll be debugging them ;)

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # testthat
 
-[![Build Status](https://travis-ci.org/hadley/testthat.png?branch=master)](https://travis-ci.org/hadley/testthat)
+[![Travis-CI Build Status](https://travis-ci.org/hadley/testthat.png?branch=master)](https://travis-ci.org/hadley/testthat) [![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/krlmlr/devtools?branch=master)](https://ci.appveyor.com/project/krlmlr/devtools)
```
Comment (krlmlr) [inline README.md]: Wrong badge URL... doh! Pull follows.

**Comment by krlmlr** [comment]
Sure!

**Pull request merged**